backend/contributions/models.py

```
@@ -81,6 +81,19 @@ class ContributionType(BaseModel):
         (REVIEW_FLOW_STANDARD, 'Standard'),
         (REVIEW_FLOW_BUILDER_PROJECT, 'Builder Project'),
     ]
+    BUILDER_CATEGORY_SLUG = 'builder'
+    BUILDER_DEFAULT_ESCALATION_THRESHOLD_POINTS = 400
+    BUILDER_REVIEW_DEFAULT_FIELDS = {
+        'requires_ai_review',
+        'escalation_threshold_points',
+    }
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)